src/platformIcon.tsx

```diff
@@ -65,8 +66,11 @@ export const PLATFORM_TO_ICON = {
   "javascript-cordova": "cordova",
   "javascript-electron": "electron",
   "javascript-ember": "ember",
+  "javascript-fastify": "fastify",
   "javascript-gatsby": "gatsby",
+  "javascript-hapi": "hapi",
   "javascript-ionic": "ionic",
+  "javascript-nestjs": "nestjs",
```

Comment:
These should be `node-fastify`, `node-hapi`, `node-nestjs` instead of `javascript-*`

Reply:
Should be resolved now!